File: app/src/main/java/co/tpcreative/supersafe/model/DriveAbout.java
package co.tpcreative.supersafe.model;
import java.io.Serializable;
import java.util.List;

import co.tpcreative.supersafe.common.api.response.BaseResponseDrive;
import co.tpcreative.supersafe.common.response.DriveResponse;

public class DriveAbout extends BaseResponseDrive implements Serializable{
    public long inAppUsed;
    public DriveUser user;
    public StorageQuota storageQuota;
    /*Create folder*/
    /*Drive api queries*/
    public List<DriveResponse> files;
}

File: app/src/main/java/co/tpcreative/supersafe/model/DriveAbout.kt
package co.tpcreative.supersafe.model
import co.tpcreative.supersafe.common.api.response.BaseResponseDrive
import co.tpcreative.supersafe.common.response.DriveResponse
import java.io.Serializable

class DriveAbout : BaseResponseDrive(), Serializable {
    var inAppUsed: Long = 0
    var user: DriveUser? = null
    var storageQuota: StorageQuota? = null

    /*Create folder*/ /*Drive api queries*/
    var files: MutableList<DriveResponse>? = null
}